source/incident-collaboration/getting-started.rst

```diff
@@ -1,22 +1,22 @@
 Getting Started 
 ===============
 
-Mattermost Incident Collaboration is included in the Mattermost Cloud workspace, enabled by default, and upgraded automatically.
+Mattermost Playbooks is included in the Mattermost Cloud workspace, enabled by default, and upgraded automatically.
```

Comment:
At /this/ moment, the product is still (awkwardly) called `Incident Collaboration`, even though most other in-product terminology has shifted. I'm thinking we want to keep the old name for now, and launch the full rebrand with the September v6.0 release.